### PR TITLE
Fix a follow up false positive in CA1062 from #2525

### PR DESCRIPTION
**Pull request by mavasani**
Fixes https://github.com/dotnet/roslyn-analyzers/issues/2525#issuecomment-503180297

### CODE REVIEW

**Comment by mavasani** [comment]
@RobSiklos

File: src/Microsoft.CodeQuality.Analyzers/UnitTests/QualityGuidelines/ValidateArgumentsOfPublicMethodsTests.cs [hunk 1]
```diff
@@ -1932,6 +1932,97 @@ private static void NoValidate(C c)
...
+    public IList<object> Items { get; }
+}
+
+public static class Issue2578Test
+{
+    public static void DoSomething(DataThing input)
+    {
+        Param.RequireNotNull(input);
+
+        // This line still generates a CA1062 error.
```
Comment (dotpaul) [inline src/Microsoft.CodeQuality.Analyzers/UnitTests/QualityGuidelines/ValidateArgumentsOfPublicMethodsTests.cs]: >still generates [](start = 21, length = 15)

Now that it's fixed, it should say "shouldn't generate", right?

Comment (mavasani) [reply src/Microsoft.CodeQuality.Analyzers/UnitTests/QualityGuidelines/ValidateArgumentsOfPublicMethodsTests.cs]: Yes, but I am normally very lazy to edit provided minimal repros :). I hope the work item link above makes it easy to reason about it, though I will make sure to strip the comments from repro code going forward. Thanks!

**Comment by RobSiklos** [comment]
@mavasani For some reason, I still get the same warning using 2.9.4-beta1.19319.2+e0ba1dd7. Is it possible I'm missing something?

**Comment by mavasani** [comment]
@RobSiklos I just installed 2.9.4-beta1.19319.2 to a fresh C# project with both the test cases added in this PR and don't see this false positive. Can you confirm you are using the latest package?

**Comment by RobSiklos** [comment]
@mavasani Sorry about that - seems like I needed to restart VS2019 after updating the NuGet package before changes truly took effect.

**Pull request merged**